common/src/rest/api.ts

```diff
@@ -0,0 +1,75 @@
+/*!
```

Comment:
I think we should consider splitting this file as well like the vRoRestClient. Now it is relatively small and simple but it might grow significantly in the future.